nef-plugin/EditorError.swift

```diff
@@ -0,0 +1,29 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+
+enum EditorError: Int, Error {
```

Comment:
Is there any need to make this an `Int` backed enum?

Reply:
yep, Xcode requires launch errors as `NSError` and we need setup an error code - you can find the use in [terminateError](https://github.com/bow-swift/nef-plugin/blob/b9060c6ab45591c2d89fe7760e2f379854dc19b2/nef-plugin/SourceEditorCommand.swift#L97)